### PR TITLE
fix CI configuration 

### PR DESCRIPTION
**Pull request by azjezz**

### CODE REVIEW

**Comment by azjezz** [comment]
[`hrtime()`](https://github.com/symfony/polyfill/blob/master/src/Php73/Php73.php#L20-L32) is failing under windows starting from [`4f37522`](https://github.com/symfony/polyfill/commit/4f37522d39c76741244899923d1ba6b47e26f6bc) ( which doesn't include any changes to the implementation itself )

**Comment by azjezz** [comment]
cc @nicolas-grekas 
see : https://github.com/symfony/symfony/pull/15661/files

**Comment by azjezz** [comment]
cc @nicolas-grekas

File: appveyor.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ install:
     - cinst wget
     - IF EXIST c:\php (SET PHP=0) ELSE (mkdir c:\php)
     - cd c:\php
-    - IF %PHP%==1 appveyor DownloadFile https://github.com/symfony/binary-utils/releases/download/v0.1/php-5.3.3-nts-Win32-VC9-x86.zip
-    - IF %PHP%==1 7z x php-5.3.3-nts-Win32-VC9-x86.zip -y >nul
+    - IF %PHP%==1 curl -fsS -o php-5.3.zip https://windows.php.net/downloads/releases/archives/php-5.3.11-nts-Win32-VC9-x86.zip
```
Comment (nicolas-grekas) [inline appveyor.yml]: please keep the current way of downloading: unless it's an issue, we shouldn't change unrelated things.

Comment (azjezz) [reply appveyor.yml]: it is an issue :confused: for some weird reason, windows.php.net responded with an html page when using `appveyor DownloadFile`, the only way i was able to get this to work was using `curl`

**Comment by nicolas-grekas** [comment]
Thank you @azjezz.

**Pull request merged**